apps/src/lib/levelbuilder/lesson-editor/ActivitySectionCardButtons.jsx

```diff
@@ -126,7 +126,7 @@ class ActivitySectionCardButtons extends Component {
         <div style={styles.bottomControls}>
           <span>
             <button
-              onMouseDown={this.handleOpenAddLevel}
+              onClick={this.handleOpenAddLevel}
```

Comment:
@dmcavoy can you remind me if there are any specific issues to check for when changing onMouseDown to onClick?

Reply:
I'm unable to see any problems in UI tests, unit tests or manual testing, so I'm going to proceed with changing this to a click event so that we can get our UI tests re-enabled in these browsers.

Reply:
Hey Dave - Sorry I was late getting back to you on this. I ran into issues where onClick was causing other onClick actions such as saving to occur when I didn't want it.

Reply:
No worries Dani. Do you remember if that was happening in UI tests, in unit tests, or in manual testing?

Reply:
I was hitting it in manual testing.